### ISSUE TITLE
Log list of stale indexes at startup

### ISSUE THREAD

**Issue by WilliamBZA** [Particular/ServiceControl#960]
It would be useful when debugging problems that customers are experiencing if we were to have a lis of stale indexes directly in the log files instead of having to talk the customers through the process of switching to maintenance mode etc.

This can be done via a simple GET at startup as shown here:
https://ravendb.net/docs/article-page/2.5/Csharp/server/administration/statistics

**Comment by WilliamBZA** [comment]
And a quick 5 minute spike shows it works ok:

```CSharp
var webRequest = System.Net.WebRequest.CreateHttp($"http://{documentStore.Configuration.HostName}:{documentStore.Configuration.Port}{documentStore.Configuration.VirtualDirectory}/stats");
using (var response = webRequest.GetResponse())
{
    using (var reader = new StreamReader(response.GetResponseStream()))
    {
        var json = reader.ReadToEnd();

        var jtoken = Newtonsoft.Json.Linq.JObject.Parse(json);
        var staleIndexes = jtoken["StaleIndexes"].ToObject<string[]>();
    }
}
```

**Comment by WilliamBZA** [comment]
Although we might want to wait a few minutes after startup before running the code, to give things a chance to settle.

**Comment by gbiellem** [comment]
@WilliamBZA  how does that work outside maint mode?  Raven HTTP stack should be inaccessible

**Comment by WilliamBZA** [comment]
Good point indeed.  And the `DatabaseStatistics` object doesn't seem to be available in 2.5, so we might have to enumerate each of the indexes manually and check their states that way.

**Comment by WilliamBZA** [comment]
Not a concern anymore.
